### PR TITLE
Modify the general introduction and the add feature to add patients

### PR DESCRIPTION
**Pull request by xhphoong**

### CODE REVIEW

**Review by ou4jairmeiogusrjejiepr — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`** **'c/P n/John p/98765432 e/john@example.com g/M a/Bishan street, block
+     123, #01-01 t/Asthma d/2022-12-12 1350`** : Adds a patient named John to Healthcare Xpress.
```
Comment (LolfoollorS) [inline docs/UserGuide.md]: I think that on line 29, the wrong ` is used which caused it to not code block correctly.

Maybe we should add a code block to John's name?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -73,19 +74,25 @@ Shows a message explaning how to access the help page.
...
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+
+* A person can have any number of tags (including 0).
+
+* Date and time need to be in the format of yyyy-MM-dd HHmm.
+
+* To add a patient, type c/P specifically.
+
```
Comment (LolfoollorS) [inline docs/UserGuide.md]: I think the > is missing for all of the text

**Review by LolfoollorS — COMMENTED** [review]
LGTM just some minor things to fix

File: docs/UserGuide.md (hunk 1 above)
Comment (xhphoong) [reply docs/UserGuide.md]: thanks for catching this mistake!

**Pull request merged**